lib/classes/Variables.test.js

```diff
@@ -688,6 +688,23 @@ describe('Variables', () => {
       });
     });
 
+    it('should populate symlinks', () => {
+      const serverless = new Serverless();
+      const SUtils = new Utils();
+      const tmpDirPath = testUtils.getTmpDirPath();
+      const realFilePath = path.join(tmpDirPath, 'someFile');
+      const symlinkPath = path.join(tmpDirPath, 'refSomeFile');
+
+      SUtils.writeFileSync(realFilePath, 'hello world');
```

Comment:
Shouldn't we remove the file and the symlink again after the test as a clean up step?